.github/workflows/release.yml

```diff
@@ -55,10 +55,28 @@ jobs:
         include:
           - arch: "x86_64"
             libc: "musl"
-          - arch: "aarch64"
-            libc: "musl"
           - arch: "i686"
             libc: "musl"
+    steps:
```

Comment:
To avoid repeating the steps, we can include another value in the matrix list above like
```
- runner: ubuntu-latest
  arch: "x86_64"
  libc: "musl"
```
and then use
```
runs-on: ${{ matrix.runner }}
```